cmd/multiclusterapp.go

```diff
@@ -101,6 +112,19 @@ func MultiClusterAppCommand() cli.Command {
 						Name:  "target,t",
 						Usage: "Target project names/ids to install the app into",
 					},
+					cli.StringSliceFlag{
+						Name:  "role",
+						Usage: "Set roles required to view the app",
+					},
+					cli.StringSliceFlag{
+						Name:  "member",
+						Usage: "Set members who can view the app, with access type defined by --member-access-type",
+					},
+					cli.StringFlag{
+						Name:  "member-access-type",
```

Comment:
Should each members have their own access type?

Reply:
It's not easy to claim different members with different access type in a single command. 
Therefore,
1. These arguments are for setting a set of members with the same access type on install.
2. We introduce add/delete-member to do accurate membership assignments